spec/fake_standard_out.rb

```diff
@@ -0,0 +1,12 @@
+class FakeStandardOut
```

Comment:
I think it is conventional to put class like this into a `spec/support` directory and load them within the `spec_helper`. However, it is possible that we may not need to move this to its own file.